applications_tests/gls_nitsche_navier_stokes_33/sphere.msh

```diff
@@ -0,0 +1,7561 @@
+$MeshFormat
```

Comment:
This seems like a very heavy mesh, does it have to be this big. Accuracy is not that important for an application test like this?

Reply:
I made it lighter and the coarse results are indeed the same